spec/models/mood_spec.rb

```diff
@@ -0,0 +1,40 @@
+# == Schema Information
+#
+# Table name: moods
+#
+#  id             :integer          not null, primary key
+#  name           :string
+#  description    :text
+#  created_at     :datetime
+#  updated_at     :datetime
+#  userid         :integer
+#  slug           :string
+#
+
+describe Mood do
+  let(:user) { create(:user1) }
+
+  describe '.add_premade' do
+    context 'new user without moods - before clicking premade' do
+      it 'has no moods attached to user' do
+        expect(user.moods).to eq([])
+        expect(user.moods.size).to eq(0)
+      end
+    end
+
+    context 'new user without moods - clicks on premade' do
+      it 'attaches 5 default premade moods to user' do
+        Mood.add_premade(user.id)
+
+        expect(user.moods.size).to eq(5)
+        expect(user.moods.pluck(:name)).to eq([
+          "Accepting",
```

Comment:
This should be fine actually! @keitheous can you rebase when you have the chance to? Should be ready to merge!

Reply:
Hello!! Thanks for the comments @julianguyen & @timorthi . You raised a valid point. I'll see what I can do this weekend about testing against the key rather than the value. I'll rebase this weekend. Also refactored some of the controller. Thanks Julian. Thanks Timothy